src/ray/common/buffer.h

```diff
@@ -152,7 +152,7 @@ class SharedMemoryBuffer : public Buffer {
 
   size_t Size() const override { return size_; }
 
-  bool OwnsData() const override { return false; }
+  bool OwnsData() const override { return true; }
```

Comment:
This looks dangerous to me! Are we sure there is no other code that depends on this being `false`? But I don't understand the code well enough to know if this is a problem or not. Maybe @stephanie-wang knows more.

Reply:
I change this because the original `PlasmaBuffer` set this to `true`. Also `SharedMemoryBuffer` is introduced by myself in a previous PR, so I think although this does not really makes any effect, it is better to keep the original value (of `PlasmaBuffer`) in case we will use it later.